pandas/tests/indexes/common.py

```diff
@@ -325,6 +372,11 @@ def test_argsort(self):
 
     def test_numpy_argsort(self):
         for k, ind in self.indices.items():
+
+            # 'mixedIndex' unorderable in Python3
+            if k in ['mixedIndex']:
```

Comment:
no add a specific test for this. (and test in both py2 & py3) (obviously need an if). This is the behavior need to nail down.